app/scripts/controllers/modalBenefit.js

```diff
@@ -15,10 +15,10 @@ angular.module('admissionSystemApp')
 // Please note that $modalInstance represents a modal window (instance) dependency.
 // It is not the same as the $modal service used above.
 
-  .controller('ModalInstanceBenefitCtrl', function ($scope, $modalInstance) {
+  .controller('ModalInstanceBenefitCtrl', function ($scope, $modalInstance, progressBarService) {
```

Comment:
Controller's filename has to have the same name as controller, like MyCustomCtrl -> myCustom.js